browser/importer/brave_profile_writer.cc

```diff
@@ -106,13 +106,13 @@ void BraveProfileWriter::SetBridge(BraveInProcessImporterBridge* bridge) {
   bridge_ptr_ = bridge;
 }
 
-void BraveProfileWriter::OnWalletInitialized(brave_rewards::RewardsService*
-  rewards_service, int error_code) {
-  if (error_code) {
+void BraveProfileWriter::OnWalletInitialized(
+    brave_rewards::RewardsService* rewards_service, int result) {
+  if (result != 0 && result != 12) {  // 12: ledger::Result::WALLET_CREATED
```

Comment:
Would be better to use the constants... but I know I ran into issues trying to do that (including the headers breaks the build)

Reply:
Although I would prefer enum here, I'm not sure we could use ledger::Result which was defined in bat-native ledger lib in other components. AFAIK, we're also using int values directly in brave_rewards_ui.cc.
I'm going to merge this PR first to fix the bug, we could probably refactor this part along with similar stuff in brave_rewards_ui.cc later.

Reply:
cc @NejcZdovc ^^

Reply:
@bridiver can tell you more about it, but we only use ledger lib inside rewards folder. For everything else we create duplicate definition inside core